server/task/task-memory/src/test/java/org/apache/james/task/SerialTaskManagerWorkerTest.java

```diff
@@ -162,17 +161,16 @@ void theWorkerShouldReportThatATaskIsInProgress() throws InterruptedException {
 
         TaskWithId taskWithId = new TaskWithId(id, inProgressTask);
 
-        worker.executeTask(taskWithId).subscribe();
+        worker.executeTask(taskWithId).subscribeOn(Schedulers.elastic()).subscribe();
 
         await(taskLaunched);
         verify(listener, atLeastOnce()).started(id);
         verifyNoMoreInteractions(listener);
         latch.countDown();
     }
 
-    @Disabled("JAMES-3172 We cannot cancel computation started by Reactor")
     @Test
-    void reactiveTasksShouldBeCancellable() throws InterruptedException {
+    void tasksCallingReactorShouldBeCancellable() throws InterruptedException {
```

Comment:
what does this test name means?

Reply:
A task calling reactor should be cancellable

Maybe `taskCallingReactorShouldDisposeReactorResourcesUponCancel` ?

Reply:
the problem is "call reactor", it's not clear to me what it means